packages/http-specs/specs/routes/main.tsp

```diff
@@ -315,7 +315,7 @@ namespace PathParameters {
       @scenarioDoc("""
         Test matrix expansion with explode: true when passed an array value.
             Param value: ["a","b"]
-            Expected path: /routes/path/matrix/explode/array;a.b
+            Expected path: /routes/path/matrix/explode/array;param=a.b
```

Comment:
is this correct?

Reply:
you are right, should be `/routes/path/matrix/explode/array;param=a;param=b`